### PR TITLE
adding encrypt command

### PR DESCRIPTION
**Pull request by mrsmkl**
## Description

Add a description of your changes here.

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

File: src/commands/utils/encrypt.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  account: Account,
+  argv: any,
+  config: ConfigEntry,
+  logger: Logger
+): Promise<ExecutionOutput> => {
+  const { file, password } = argv
+  const encrypted = fs.readFileSync(file).toString('binary')
+  const decrypted = aes_encryption_256(encrypted, password)
+
+  const filePathEncrypted = file + '.encrypted'
```
Comment (aaitor) [inline src/commands/utils/encrypt.ts:27]: Similar to other commands here we can add an intro message saying what we are doing

**Review by aaitor — COMMENTED** [review]
It looks good, probably missing a test encrypting and decrypting a file

**Review by aaitor — APPROVED** [review]
looking good

File: resources/commands.json [hunk 2]
```diff
@@ -1230,6 +1230,23 @@
 					"default": "",
 					"description": "The password to be used to decrypt the file"
 				}]
+			}, {
+				"name": "encrypt [file]",
+				"description": "Encrypt a file",
+				"details": "It encrypts a previously encrypted file using the password provided.",
```
Comment (r-marques) [inline resources/commands.json]: I think the details are wrong

File: src/commands/utils/decrypt.ts [hunk 3]
```diff
@@ -13,12 +14,13 @@ export const decryptFile = async (
   config: ConfigEntry,
   logger: Logger
 ): Promise<ExecutionOutput> => {
+  logger.info(chalk.dim(`Encrypting file`))
```
Comment (r-marques) [inline src/commands/utils/decrypt.ts]: This should be Decrypting file, right?

File: src/commands/utils/encrypt.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+export const encryptFile = async (
+  nvm: Nevermined,
+  account: Account,
+  argv: any,
+  config: ConfigEntry,
+  logger: Logger
+): Promise<ExecutionOutput> => {
+  logger.info(chalk.dim(`Encrypting file`))
+  const { file, password } = argv
+  const encrypted = fs.readFileSync(file).toString('binary')
```
Comment (r-marques) [inline src/commands/utils/encrypt.ts]: this should be `decrypted` or `plaintext`

File: src/commands/utils/encrypt.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  const decrypted = aes_encryption_256(encrypted, password)
```
Comment (r-marques) [inline src/commands/utils/encrypt.ts]: this should be `encrypted`

File: test/integration-tests/EncryptDecrypt.test.ts [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  })
+
+  test('Decrypt the content', async () => {
+    const decryptCommand = `${baseCommands.utils.decrypt} --password ${password}  README.md.encrypted`
+    console.debug(`COMMAND: ${decryptCommand}`)
+
+    const decryptStdout = execCommand(decryptCommand, execOpts)
+    console.log(`STDOUT: ${decryptStdout}`)
+
+    expect(decryptStdout.includes(`File decrypted successfully`))
```
Comment (r-marques) [inline test/integration-tests/EncryptDecrypt.test.ts:25]: should we also check that the decrypted file content is equal to the content of `README.md`

File: resources/commands.json [hunk 7]
```diff
@@ -1230,6 +1230,23 @@
...
+					"name": "file",
+					"type": "string",
+					"description": "The path to the file to encrypt"
+				}],
+				"optionalArguments": [{
+					"name": "password",
+					"type": "string",
+					"default": "",
+					"description": "The password to be used to encrypt the file"
+				}]
```
Comment (r-marques) [inline resources/commands.json]: I don't think password should be an option argument

**Comment by aaitor** [comment]
@mrsmkl , can you confirm if comments were addressed please?

**Comment by mrsmkl** [comment]
Looks like there's still some debugging left with tests, I'll work on it tomorrow

**Pull request merged**